build/azure-pipelines-nightly.yml

```diff
@@ -6,248 +6,235 @@ trigger: none
 # No Pull request (PR) triggers for nightly
 pr: none
 
-# Nightly using schedules
+# Nightly using schedule
 schedules:
 - cron: "0 0 * * Mon,Tue,Wed,Thu,Fri"
   displayName: midnightly build
   branches:
     include:
     - main
 
-pool:
-  vmImage: 'windows-2022'
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: refs/heads/main
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
 
 variables:
```

Comment:
I think you're missing some variables from the other pipelines...I'm not sure whether it matters though : 

variables:
- name: __NugetSecurityAnalysisWarningLevel__
  value: none
- name: runCodesignValidationInjection
  value: false

Reply:
I thought of adding them but I wasn't sure whether they're strictly necessary especially because they appear to be assigned default values. I'll evaluate whether they're necessary when updating the template to use .NET 8.0

Reply:
Actually, those variables are not applied anywhere even in the other pipelines.